ceph-rbd-mirror/src/tests/target.py

```diff
@@ -288,7 +288,7 @@ def wait_for_mirror_state(self, state, application_name=None,
         :type pools: list of str
         :returns: True on success, never returns on failure
         """
-        rep = re.compile(r'.*"entries_behind_primary":(\d+),')
+        rep = re.compile(r'.*"entries_behind_master":(\d+),')
```

Comment:
FWIR (from ceph-rgw) we moved away from the master/slave nomenclature to primary/secondary nomenclature.

Reply:
This is what the charm logs and what we have to match. We can change that, but it would change the charm and thus we need to re-release it.

Reply:
ack, added a ticket to resolve such occurrences in batches.